portfolio/src/main/webapp/index.html

```diff
@@ -98,10 +96,17 @@ <h3 id="contact_me">Contact Me</h3>
       <a href="https://www.github.com/KathyFeiyang">
         <img src="/images/GitHub_icon.png" width="50" height="50">
       </a>
-      <p><b>Submit feedback :)</b></p>
-      <form id="feedback-form" onsubmit="presentFeedbackReceipt()" action="/index.html#contact_me">
+      <p><b>Comment History</b></p>
```

Comment:
I'd maybe move this to below the "submit your comments" section (seems more in line with general internet conventions)